### DISCUSSION TITLE
add new constraints to my model

### DISCUSSION THREAD

**Discussion by BYS543** [yalmip/YALMIP#936] [Optimization modelling] [answered]
Hi,Johan
    can I do some thing like this in Yalmip?
```
    generate a model;
    for i=1:N
        add(model,Constraint(i));   %add constraint to the established model during the iteration process
    end
```
Because i don't want to create a model repeatly, i just want to adjust my model by adding some constraints.

**Comment by johanlofberg** [comment, marked as answer]
```
   for i=1:N
        model = [model, Constraint(i)]
    end
```

https://yalmip.github.io/tutorial/basics/

**Reply by BYS543** [reply]
thank you,Johan. I just want to save some time for compiling my codes faster.

**Reply by johanlofberg** [reply]
The only way to compile faster is to use optimizer for parameter changes https://yalmip.github.io/command/optimizer/ (which you can use to turn on/off constraints)
